pkg/controllers/managementuser/secret/secret.go

```diff
@@ -250,12 +250,14 @@ func getNamespacedSecret(obj *corev1.Secret, namespace string) *corev1.Secret {
 	namespacedSecret.Namespace = namespace
 	namespacedSecret.Type = obj.Type
 	namespacedSecret.Annotations = make(map[string]string)
-	copyMap(obj.Annotations, namespacedSecret.Annotations)
+	namespacedSecret.Labels = make(map[string]string)
+	copyMap(namespacedSecret.Annotations, obj.Annotations)
+	copyMap(namespacedSecret.Labels, obj.Labels)
 	namespacedSecret.Annotations[userSecretAnnotation] = "true"
 	return namespacedSecret
 }
 
-func copyMap(src map[string]string, dst map[string]string) {
+func copyMap(dst map[string]string, src map[string]string) {
```

Comment:
Yes, it's not strictly necessary. Flipping it brings it in line with the loose Go convention of specifying the destination as the first parameter of such functions. Especially since it works with values that contain pointers internally. Like the builtin `copy`, for example.

Reply:
Hmm okay, that change sounds good to me then